src/Components/WebAssembly/Authentication.Msal/src/Interop/webpack.config.js

```diff
@@ -1,25 +1,72 @@
 const path = require('path');
+const webpack = require('webpack');
+const TerserJsPlugin = require("terser-webpack-plugin");
+const { DuplicatesPlugin } = require("inspectpack/plugin");
 
-module.exports = env => {
-
-    return {
-        entry: './AuthenticationService.ts',
-        devtool: env && env.production ? undefined : 'eval-source-map',
-        module: {
-            rules: [
-                {
-                    test: /\.tsx?$/,
-                    use: 'ts-loader',
-                    exclude: /node_modules/,
+module.exports = (env, args) => ({
+    resolve: {
+        extensions: ['.ts', '.js'],
+    },
+    devtool: false, // Source maps configured below
+    module: {
+        rules: [{ test: /\.ts?$/, loader: 'ts-loader' }]
+    },
+    entry: {
+        'AuthenticationService': './AuthenticationService.ts',
+    },
+    output: { path: path.join(__dirname, '/dist', args.mode == 'development' ? '/Debug' : '/Release'), filename: '[name].js' },
```

Comment:
```suggestion
    output: { path: path.join(__dirname, 'dist', args.mode == 'development' ? 'Debug' : 'Release'), filename: '[name].js' },
```

`path.join` already inserts the correct platform-specific separators.